integration-cli/docker_cli_help_test.go

```diff
@@ -15,7 +16,13 @@ func TestMainHelpWidth(t *testing.T) {
 
 	var home string
 	if runtime.GOOS != "windows" {
-		home = os.Getenv("HOME")
+		// Only do this on non-windows systems
+		home = homedir.Get()
+
+		// Don't check when HOME is /
+		if home == "/" {
+			home = ""
```

Comment:
Ah, good point.  Instead of just testing the current env's $HOME I should test / too.  I modified merged the two tests and looped it now so that on non-windows we'll test the current $HOME and /.  See what you think.